modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/misc/GithubApi2.kt

```diff
@@ -110,38 +114,42 @@ class GithubApi2 @Inject constructor(
         get() {
             val callable = Callable<String> {
                 var result = "0"
-
-                val username = localProperties.getNoThrows(PROPERTY_USERNAME, DOC_URL)
-                val accessToken = localProperties.getNoThrows(PROPERTY_ACCESS_TOKEN, DOC_URL)
-                try {
-                    val req =
-                            if (username != null && accessToken != null) {
-                                service.getReleases(username, "kobalt", accessToken)
-                            } else {
-                                service.getReleasesNoAuth("cbeust", "kobalt")
-                            }
-                    val ex = req.execute()
-                    val errorBody = ex.errorBody()
-                    if (errorBody != null) {
-                        val jsonError = JsonParser().parse(errorBody.string())
-                        warn("Couldn't call Github.getReleases(): $jsonError")
-                    } else {
-                        val releases = ex.body()
-                        if (releases != null) {
-                            releases.firstOrNull()?.let {
-                                try {
-                                    result = listOf(it.name, it.tagName).filterNotNull().first { !it.isBlank() }
-                                } catch(ex: NoSuchElementException) {
-                                    throw KobaltException("Couldn't find the latest release")
+                if (Duration.ofMinutes(10L) >
```

Comment:
Might be a bit annoying for me and other Kobalt developers, but we can look into making this configurable later.